src/adapter/etl-adapter-json/src/Flow/ETL/Adapter/JSON/JSONMachine/JsonlExtractor.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\Adapter\JSON\JSONMachine;
+
+use function Flow\ETL\DSL\{array_to_rows};
+use Flow\ETL\Extractor\{FileExtractor, Limitable, LimitableExtractor, PathFiltering, Signal};
+use Flow\ETL\Row\Schema;
+use Flow\ETL\{Extractor, FlowContext};
+use Flow\Filesystem\{Path};
+use JsonMachine\Items;
+use JsonMachine\JsonDecoder\ExtJsonDecoder;
+
+final class JsonlExtractor implements Extractor, FileExtractor, LimitableExtractor
```

Comment:
I think I would call it JsonLineExtractor, seems to be more self descriptive than just JsonL, wdyt?

Reply:
definitely better